src/assets/images/carot-down-picker.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="8" height="6" viewBox="0 0 8 6" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
is this asset a duplicate of the one below it?

Reply:
Yeah, I couldn't get it to work in the picker with a custom color so I just made one specifically for the picker since we use that color in multiple places and then I made another where you can change the color. Not sure if there is a better way?

Reply:
ah ok yeah idk, might have to change something about the picker. this is fine